assets/js/leaves.js

```diff
@@ -126,25 +172,48 @@ class Leaf {
         // actually make selection
         Leaf.setLeafLabelClass(leafTarget.dataset.url, Leaf.selectedClass);
```

Comment:
So now only one leaf is 'selected' at a time? And if a tag is clicked, those leaves are 'highlighted' rather than 'selected' ?

Reply:
Yes, when I looked back at the styles I remembered that there was a difference in display — the leaf that is active and has details shown in the panel has an outline; the others have the highlight glow but no outline.  Does highlight / select seem like a good way to distinguish these?

Reply:
Yep I think so! Just clarifying to myself. I'm glad we're making this distinction